### PR TITLE
tewisay: init at git-2017-04-14

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change
I like cowsay, so I wanted to add a fun [alternative](https://github.com/lucy/tewisay) to this repository.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by nlewo** [comment]
Tested and LGTM.

**Comment by Chiiruno** [comment]
Is there anything I need to do for this to be merged, or is it good as it is?

File: pkgs/tools/misc/tewisay/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "tewisay-unstable-${version}";
```
Comment (nlewo) [inline pkgs/tools/misc/tewisay/default.nix]: Could you remove "unstable"?

Comment (nlewo) [reply pkgs/tools/misc/tewisay/default.nix]: Sorry, that's good like this in fact... (https://nixos.org/nixpkgs/manual/#sec-package-naming)

**Review by nlewo — APPROVED** [review]

**Comment by nlewo** [comment]
@Mic92 LGTM. Could you have a look please?

**Comment by Mic92** [comment]
Thanks!

**Comment by Chiiruno** [comment]
Thank you!

**Pull request merged**